<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClientDialog</class>
 <widget class="QDialog" name="ChatClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatClient</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sendMsg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YaHei Consolas Hybrid</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ServerAddress:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addressEdit">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ServerPort:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portEdit">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8080</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_Connect">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Disconnect">
      <property name="font">
       <font>
        <family>YaHei Consolas Hybrid</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DisConnect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
